docs/whats-new.mdx

```diff
@@ -105,6 +105,10 @@ Target Release Date: April, 2023.
 - [`BSONLoader`](/docs/modules/bson/api-reference/bson-loader) - loads BSON encoded binary data into a "JSON like" JavaScript structure
 - [`BSONWriter`](/docs/modules/bson/api-reference/bson-writer) - writes a JSON like JavaScript structure into a BSON encoded binary data block.
 
+**@loaders.gl/gis**
+
+- `triangulate` parameter added to `geojsonToBinary()`.
```

Comment:
@ibgreen as suggested, I will backport to 3.4 after merging to master

Reply:
All of the above sounds good, thanks, please go ahead.